bam-to-unmapped-bams.wdl

```diff
@@ -75,49 +62,27 @@ workflow BamToUnmappedBams {
   }
 }
 
-task GenerateOutputMap {
-  File input_bam
-  Int disk_size
-  
-  String docker
-
-  command {
-    set -e
-
-    samtools view -H ${input_bam} | grep @RG | cut -f2 | sed s/ID:// > readgroups.txt
-
-    echo -e "READ_GROUP_ID\tOUTPUT" > output_map.tsv
-
-    for rg in `cat readgroups.txt`; do
-      echo -e "$rg\t$rg.coord.sorted.unmapped.bam" >> output_map.tsv
-    done
-  }
-
-  runtime {
-    docker: docker
-    disks: "local-disk " + disk_size + " HDD"
-    preemptible: "3"
-    memory: "1 GB"
-  }
-  output {
-    File output_map = "output_map.tsv"
-  }
-}
-
 task RevertSam {
-  File input_bam
-  File output_map
-  Int disk_size
-
-  String gatk_path
-
-  String docker
+  input {
+    #Command parameters
+    File input_bam
+    String gatk_path
+
+    #Runtime parameters
+    Int disk_size
+    String docker
+    Int machine_mem_gb = 2
+    Int preemptible_attempts = 3
+  }
+    Int command_mem_gb = machine_mem_gb - 1    ####Needs to occur after machine_mem_gb is set 
 
-  command {
-    ${gatk_path} --java-options "-Xmx1000m" \
+  command { 
```

Comment:
I believe the `command <<<  >>>` with `~{ }` style is preferred over `command { }` with `${ }` as it frees up `$` from dual meanings of wdl placeholder and bash variable, which makes the code less confusing and bug prone.

Reply:
done